m0t1fs/linux_kernel/st/m0t1fs_rpc_cancel_test.sh

```diff
@@ -146,11 +146,16 @@ rcancel_post()
 
 rcancel_change_controller_state()
 {
-	local lnet_nid=`sudo lctl list_nids | head -1`
-	local s_endpoint="$lnet_nid:12345:33:1"
-	local c_endpoint="$lnet_nid:$M0HAM_CLI_EP"
+	local nid=`sudo lctl list_nids | head -1`
```

Comment:
Please use 
```
local nid=$(m0_local_nid_get)
```
instead of
```
local nid=`sudo lctl list_nids | head -1`
```
as `lctl list_nids` will only work for lnet.

Reply:
done